public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -401,6 +414,19 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
     );
   }
 
+  private startTrackingScopes() {
+    this._scopesTrackerSub = this.subscribeToEvent(
+      SceneObjectStateChangedEvent,
+      (event: SceneObjectStateChangedEvent) => {
+        if (event.payload.changedObject instanceof ScopeSelectorScene) {
+          if (Object.prototype.hasOwnProperty.call(event.payload.partialUpdate, 'value')) {
+            sceneGraph.getTimeRange(this).onRefresh();
+          }
+        }
+      }
+    );
+  }
```

Comment:
think we should just subscribe to  scopes selector state changes only rather than setting event subscription to object state changes. Sth lik:

```ts
this._scopesTrackerSub =  this.state.scopeSelector!.subscribeToState(...);
```

Reply:
Unfortunately I ended up in some weird edge case. Let's sync on this and see if we can solve it.